servant-client/servant-client.cabal

```diff
@@ -12,7 +12,7 @@ license:             BSD3
 license-file:        LICENSE
 author:              Servant Contributors
 maintainer:          haskell-servant-maintainers@googlegroups.com
-copyright:           2014-2016 Zalora South East Asia Pte Ltd, Servant Contributors
+copyright:           2014-2017 Zalora South East Asia Pte Ltd, Servant Contributors
```

Comment:
does zalora people have copyright over 2017?

Reply:
I can't say I understand this. Happy either way.

Reply:
2014-2016 Zalora, 2016-2017 Servant Contributors imho is closer to truth (not sure about the transition year)